src/sycl/chunked_prefill.cpp

```diff
@@ -684,6 +693,9 @@ std::vector<at::Tensor> mha_fwd(
 
   // Set the different scale values.
   params.scale_softmax = softmax_scale;
+  bool use_sink = softmax_sink_.has_value();
+  params.sink_softmax = use_sink ? static_cast<void*>(softmax_sink_.value().data_ptr()) : nullptr;
```

Comment:
return type of `data_ptr()` is just void*, you don't have to use static_cast here.

data_ptr<T>() returns T*